spec/services/mailchimp_subscription_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe MailchimpSubscription do
+  let(:email) { 'kevin@fizbaz.com' }
+  let(:username) { 'kevin' }
+  let(:signup_date) { Date.new }
+  let(:options) do
+    {
+      email: email,
+      username: username,
+      signup_date: signup_date
+    }
+  end
+  let(:list_id) { ENV['MAILCHIMP_LIST_ID'] }
+
+  describe '.create' do
+    it 'creates and returns a new instance', :vcr do
```

Comment:
I don't think checking what type of class an instance is necessary. It feels like an implementation detail instead of behaviour of the class. But what do you think?

Reply:
I removed the test. Returning an instance was just an instinct from calling `#create` on a class like with ActiveRecord. Neither the return value nor the test was extremely necessary though.